core/tests/protractor/learnerDashboard.js

```diff
@@ -210,14 +207,14 @@ describe('Learner dashboard functionality', function() {
     // Subscribe to both the creators.
     browser.get('/profile/creator1learnerDashboard');
     browser.waitForAngular();
-    element(by.css('.protractor-test-subscription-button')).click();
+    learnerDashboardPage.clickSubscriptionButton();
```

Comment:
This name is good -- but this isn't the learnerDashboardPage, it's the profilePage, right?

Reply:
ok, thanks for the suggestions, will make the necessary changes.